hotel/models/hotel.py

```diff
@@ -135,6 +134,8 @@ class HotelRoomAmenities(models.Model):
     rcateg_id = fields.Many2one('hotel.room.amenities.type',
                                 'Amenity Catagory')
 
+    product_manager = fields.Many2one('res.users', string='Product Manager')
+
```

Comment:
Sir It is amenity manager that manager of amenities & added field in old version so I add that field